lib/train/transports/ssh.rb

```diff
@@ -193,6 +193,34 @@ def verify_host_key_option
       current_net_ssh >= new_option_version ? :verify_host_key : :paranoid
     end
 
+    # Likewise, version <5 accepted false; 5+ requires :never or will
+    # issue a deprecation warning. This method allows a lot of common
+    # things through.
+    def verify_host_key_value(given)
+      current_net_ssh = Net::SSH::Version::CURRENT
+      new_value_version = Net::SSH::Version[5, 0, 0]
+      if current_net_ssh >= new_value_version
+        # 5.0+ style
+        {
+          # It's not a boolean anymore.
+          'true' => :always,
+          'false' => :never,
+          true => :always,
+          false => :never,
+          # May be correct value, but strings from JSON config
```

Comment:
Missing the nil case here, since it's possible for someone to specify the option w/ nil value.

Reply:
Done, thanks!